src/systemd/Makefile.am

```diff
@@ -25,10 +25,10 @@ dist_motd_SCRIPTS = src/systemd/update-motd
 
 # Automake: 'Variables using ... ‘sysconf’ ... are installed by install-exec.'
 install-exec-hook::
-	mkdir -p $(DESTDIR)$(sysconfdir)/motd.d
-	ln -sTfr $(DESTDIR)/run/cockpit/motd $(DESTDIR)$(sysconfdir)/motd.d/cockpit
-	mkdir -p $(DESTDIR)$(sysconfdir)/issue.d
-	ln -sTfr $(DESTDIR)/run/cockpit/motd $(DESTDIR)$(sysconfdir)/issue.d/cockpit.issue
+	mkdir -p $(DESTDIR)$(prefix)$(sysconfdir)/motd.d
```

Comment:
The tests fail for CentOS/Fedora, as sysconfdir is already prefixed.

```
        sysconfdir:                 ${prefix}/etc
```

Reply:
Weird 

```

               Cockpit 287
             ================

        prefix:                     /nix/store/nbdvnch6cwyk91x350xl80kvr8p82l6k-cockpit-287
        exec_prefix:                ${prefix}
        libdir:                     ${exec_prefix}/lib
        libexecdir:                 ${exec_prefix}/libexec
        bindir:                     ${exec_prefix}/bin
        sbindir:                    ${exec_prefix}/sbin
        datarootdir:                ${prefix}/share
        datadir:                    ${datarootdir}
        sysconfdir:                 /etc
        localstatedir:              ${prefix}/var
        pamdir:                     ${libdir}/security
        systemd unit dir:           ${prefix}/lib/systemd/system

        compiler:                   gcc
        cflags:                     -g -O2 -g
        cppflags:                   

        cockpit-ws user:            root
        cockpit-ws group:           
        cockpit-ws instance user:   root
        cockpit-ws instance group:  
```

Reply:
I found the problem. When the program runs it uses hardcoded $sysconfdir to look for stuff but $sysconfdir will be inside the nix store so it crashes.